app/build.gradle

```diff
@@ -95,6 +95,7 @@ dependencies {
     testImplementation "org.assertj:assertj-core:$assertJVersion"
     testImplementation "io.mockk:mockk:$mockkVersion"
     testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:$coroutinesVersion"
+    implementation 'com.google.android.material:material:1.0.0'
```

Comment:
sorry I didn't see it early but could you put this line close to the appcompat dependency L84.
this place is for test dependency like mockk, assertJ ...

Reply:
@CedrickFlocon Done